header/serde.go

```diff
@@ -64,37 +64,6 @@ func UnmarshalExtendedHeader(data []byte) (*ExtendedHeader, error) {
 	return out, nil
 }
 
-func ExtendedHeaderToProto(eh *ExtendedHeader) (*header_pb.ExtendedHeader, error) {
-	pb := &header_pb.ExtendedHeader{
-		Header: eh.RawHeader.ToProto(),
-		Commit: eh.Commit.ToProto(),
-	}
-	valSet, err := eh.ValidatorSet.ToProto()
-	if err != nil {
-		return nil, err
-	}
-	pb.ValidatorSet = valSet
-	dah, err := eh.DAH.ToProto()
-	if err != nil {
-		return nil, err
-	}
-	pb.Dah = dah
-	return pb, nil
-}
-
-func ProtoToExtendedHeader(pb *header_pb.ExtendedHeader) (*ExtendedHeader, error) {
```

Comment:
Wait, how do we serialize then?

Reply:
We serialise by MarshalExtendedHeader/UnmarshalExtendedHeader. Those are converter fucntions, that are not really needed